src/main/java/nl/knaw/dans/ingest/core/service/mapper/mapping/FileElement.java

```diff
@@ -170,7 +170,7 @@ static Map<String, List<String>> getKeyValuePairs(Node node, String filename, St
     }
 
     static String replaceForbiddenCharactersInPath(String dirPath) {
-        if (dirPath == null) {
+        if (dirPath == null || dirPath.isBlank()) {
```

Comment:
Before this change the github build failed on `SwordExamplesTest`, after this change `FileElementTest` fails.

Reply:
I think StringUtils.isBlank will do this without the `||`.

Reply:
Outdated code, it was reverted.